docs/env_variables.md

```diff
@@ -104,7 +104,7 @@ Then there are a few specific features that are only available in llvm_mode:
     - Setting AFL_LLVM_LAF_SPLIT_COMPARES will split all floating point and
       64, 32 and 16 bit integer CMP instructions
 
-    See llvm_mode/README.laf-intel.md for more information. 
+    See llvm_mode/README.laf-intel.md for more information.
```

Comment:
->
See [llvm_mode/README.laf-intel.md](../llvm_mode/README.laf-intel.md) for more information.

Reply:
that was one we actually overlooked :)